nuitka/tools/profiler/__main__.py

```diff
@@ -79,6 +79,7 @@ def main():
                 raise
         vmprof.disable()
 
+        # pylint: disable=E1123
```

Comment:
Also answering your question, you should install pylint via requirements-devel.txt to have the version of PyLint that handles Nuitka without the regressions.

Reply:
mmmm, so maybe the fact I'm getting a different outcome than CI is because using a different pylint version? I hadn't thought about it, good point, ill check it out